packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx

```diff
@@ -104,13 +110,37 @@ export default function useTdpClientCanvas(props: Props) {
     setWsConnection('open');
   };
 
+  const platform = navigator.userAgentData?.platform || navigator.platform;
+  const isMacOS = /mac/i.test(platform);
```

Comment:
we have this function: https://github.com/gravitational/webapps/blob/master/packages/design/src/theme/utils/getPlatform.js